assets/style.css

```diff
@@ -19,6 +19,8 @@
 }
 
 body {
+	color: #222;
+	background-color: white;
```

Comment:
I’m okay making this change, but when I tested it (or e.g. `background-color: red`) in the Chrome inspector, I don’t see any changes? Have you verified that this is the correct place to make changes?

Reply:
I've done tests on firefox web developer. And I see changes (at the bottom of the page) even for the red color.

Reply:
And in chromium too:
![image](https://user-images.githubusercontent.com/14617699/66373092-76e8e080-e9b0-11e9-954e-e52003fdc96c.png)

Reply:
Ah, this only affects the overflow area. Yeah, that works for me, too, thanks.